docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: docs
+title: Browsers
+permalink: /documentation/concepts/browsers.html
+---
+# Browsers
+
+This topic lists browsers supported by TestCafe and describes how to use various browser features.
+
+* [Browser Support](#browser-support)
+* [Test in Headless Mode](#test-in-headless-mode)
+* [Use Chromium Device Emulation](#use-chromium-device-emulation)
+* [User Profiles](#user-profiles)
+
+## Browser Support
+
+### Officially Supported Browsers
+
+While TestCafe is designed to support most modern browsers, there are a limited number
+of *officially supported* browsers against which TestCafe is actively tested.
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (11+)
+* Microsoft Edge (legacy and Chromium-based)
+* Mozilla Firefox
+* Safari
+* Google Chrome mobile
+* Safari mobile
+
+> TestCafe supports the latest version of each browser unless specified explicitly.
```

Comment:
Let's say:
> TestCafe supports the latest version of each browser (unless specified).

Reply:
Because for some browsers we support two versions (Edge) or dev versions (Chrome). In this case, we specify these versions in parentheses in the list above.

Reply:
Okay, let's say:
TestCafe supports the latest version of each browser (unless specified explicitly).